components/edit-collective/sections/Webhooks.js

```diff
@@ -295,60 +287,49 @@ class Webhooks extends React.Component {
               }),
             }}
           />
-        </P>
+        </p>
 
-        <div>{webhooks.map(this.renderWebhook)}</div>
+        <Separator className="my-6" />
 
-        {webhooksCount > 0 && <StyledHr borderColor="black.300" />}
-
-        <Box width={[0.9, 0.75]} mx={['auto', 0]} my={3}>
-          <StyledButton
-            width={1}
-            px={[0, 3, 0]}
-            borderRadius={6}
-            buttonSize="medium"
-            buttonStyle="standard"
-            css={'border-style: dashed'}
-            onClick={() => this.addWebhook()}
-          >
-            <Add size="1.2em" />
-            {'  '}
+        <div className="mb-6 mt-8 flex items-center justify-between">
+          <h3 className="text-xl font-bold text-gray-900">
```

Comment:
The shadcn/ui components is using the `slate` version of grays, and the default text color applied to all elements is `text-slate-950`.

So we could either do
```suggestion
          <h3 className="text-xl font-bold">
```
as 950 is close to 900, or 

```suggestion
          <h3 className="text-xl font-bold text-slate-900">
```
to be specific.

(We might want to remove the regular `gray`s from the available palette, or map them to the `slate` palette..)